dataplane/api-server/src/lib.rs

```diff
@@ -42,23 +44,37 @@ pub async fn start(
     let healthchecks = tokio::spawn(async move {
         let (_, health_service) = tonic_health::server::health_reporter();
         let mut server_builder = Server::builder();
-        server_builder
+
+        let port = port + 1;
+        let addr = SocketAddrV4::new(addr, port);
+        let server = server_builder
             .add_service(health_service)
-            .serve(SocketAddrV4::new(addr, port + 1).into())
-            .await
-            .unwrap();
+            .serve(addr.into());
+
+        debug!("gRPC Health Checking service listens on port {}", port);
+        server.await.map_err(|e| {
+            error!("Failed serve gRPC Health Checking service, err: {:?}", e);
+            e
+        }).unwrap();
     });
 
     // Secure server with (optional) mTLS
     let backends = tokio::spawn(async move {
         let server = server::BackendService::new(backends_map, gateway_indexes_map, tcp_conns_map);
+
         let mut server_builder = Server::builder();
         server_builder = setup_tls(server_builder, &tls_config).unwrap();
-        server_builder
+
+        let tls_addr = SocketAddrV4::new(addr, port);
+        let tls_server = server_builder
             .add_service(BackendsServer::new(server))
-            .serve(SocketAddrV4::new(addr, port).into())
-            .await
-            .unwrap();
+            .serve(tls_addr.into());
+
+        debug!("TLS server listens on port {}", port);
```

Comment:
Similar to above, what if we drop the debug at the top and just log `{addr}:{port}` together?